scripts/common/preflights.sh

```diff
@@ -233,3 +234,15 @@ function kotsadm_prerelease() {
         fi
     fi
 }
+
+function host_nameservers_reachable() {
+    if [ "$AIRGAP" = 1 ]; then
+        return 0
+    fi
+    if ! discover_non_loopback_nameservers; then
+        printf "\n${YELLOW}No nameservers configured on the host are accessible by pods. All DNS queries outside the cluster will fail. Continue?${NC} "
+        if ! confirmN; then
+            bail "\nAt least one nameserver must be accessible on a non-loopback address"
```

Comment:
"accessible".  Does this mean accessibility or configured?

Reply:
The address in resolv.conf must be non-loopback. It's about what address the client uses not what address the nameserver binds.